src/Routers.tsx

```diff
@@ -130,6 +131,7 @@ const Routes: React.FC = () => {
       <Route path={routers.TERMS_OF_SERVICE} exact component={TermOfServices} />
       <Route path={routers.NATIVE_SCRIPT_DETAIL} exact component={NativeScriptsDetailPage} />
       <Route path={routers.DREP_DETAILS} exact component={DrepDetail} />
+      <Route path={routers.BOLNISI_LANDING} exact component={BolnisiLanding} />
```

Comment:
From my point of view Bolnisi page should be in a seperate menu for use cases. Just noting it here, I will create a ticket for this.

Reply:
yahh. you can create ticket for this and i will fix it in sprint 14